src/train.js

```diff
@@ -193,7 +206,20 @@ function updateProgress(ratio, bestSolution, bestCost, successesOrFailures) {
  * Draw and outfit the Train page.
  */
 async function initPage(document) {
-    document.getElementById('train').onclick = trainOnTabs;
+    // Find elements once.
+    g_progressBar = document.getElementById('progress');
+    g_coeffs_div = document.getElementById('coeffs');
+    g_accuracy_div = document.getElementById('accuracy');
+    g_goodBadDiv = document.getElementById('goodBad');
+
+    // Initialise elements to a known state.
```

Comment:
Shouldn't they always be empty to begin with? My intention was for initPage() to run only on page load.